client/openocd_rpc.py

```diff
@@ -141,10 +146,17 @@ def read_word(self, address):
         with self._lock:
             return self._maybe_retry(self._read_word_locked, address)
 
+    def _should_ignore(self, line):
+        line = line.decode('utf-8')
```

Comment:
Maybe add `errors='ignore'`, otherwise an invalid unicode sequence will blow this up.